Podfile

```diff
@@ -5,13 +5,14 @@ target 'Fitness' do
   # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
   use_frameworks!
   pod 'AlamofireImage'
+  pod 'Apollo', :git => 'https://github.com/apollographql/apollo-ios.git', :commit => 'b28c3dc'
+  pod 'Bartinter'
   pod 'Crashlytics'
+  pod 'FLEX', '~> 2.0', :configurations => ['Debug']
```

Comment:
Do we wanna push this to master/should we learn to use this on all pods?

Reply:
its fine to have in master b/c it'll be ignored when you run/archive on Release. i find it useful b/c i don't know what anything is inside the app